pyqtgraph/parametertree/parameterTypes/pen.py

```diff
@@ -174,20 +174,13 @@ def _makeChildren(self, boundPen=None):
             name = name.title().strip()
             p.setOpts(title=name, default=default)
 
-        def penPropertyWrapper(propertySetter):
-            def tiePenPropToParam(_, value):
-                propertySetter(value)
-                self.sigValueChanging.emit(self, self.pen)
-
-            return tiePenPropToParam
-
         if boundPen is not None:
             self.updateFromPen(param, boundPen)
             for p in param:
-                setter, setName = self._setterForParam(p.name(), boundPen, returnName=True)
+                _, setName = self._setterForParam(p.name(), boundPen, returnName=True)
```

Comment:
I recommend removing `_setterForParam` entirely since it's only used here and simply doing `setName = f'set{p.name().capitalize()}'

This is the only place it's used, and it's an internal method, so there should be no problems with its removal